docs/api/index-api.rst

```diff
@@ -28,6 +28,9 @@ Submodules
     * - :mod:`models. <datatable.models>`
       - A small set of data analysis tools.
 
+    * - :mod:`time. <datatable.time>`
```

Comment:
Why not `datatable.datetime`?

Reply:
I use the word "time" not in the  sense of "time of day", but as the physical _time_. That thing that keeps everything from happening at once. The fourth dimension.

For me, from a physicist's standpoint, the word "time" is perfectly self-sufficient, and doesn't need to be contorted into a "datetime" or "timestamp" or "time_point" or whatever. Specifically, a "date" is a subset of time -- 24-hour long slices of time to be precise, and the module name `dt.time` is thus suitable for describing everything temporal.

Plus, it's shorter and easier to type.